src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -1063,5 +1063,65 @@ public void BlockDataDownloadedIsCalled_ForValidBlocksAfterFv_ResultShouldBeTrue
             resultForH5.Should().BeTrue();
             chainTipH5.Block.Should().NotBeNull();
         }
+
+        /// <summary>
+        /// Issue 26 @  Checkpoints are disabled. Chain tip is at header 5. Reorganisation is initiaised with max reorg of 500 blocks plus extra 50 blocks. 
```

Comment:
It is supposed to be 27

Reply:
And there actually is no reorganisation in this test, it is more about discarding a chain that can no longer be viable in the future.

Reply:
Or more correctly, it is about identifying the peers that presents such chains.